pom.xml

```diff
@@ -206,22 +208,22 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>3.0.0-M5</version>
       </plugin>
-      <plugin>
-        <groupId>com.coveo</groupId>
-        <artifactId>fmt-maven-plugin</artifactId>
-        <version>2.13</version>
-        <configuration>
-          <filesNamePattern>.*\.java</filesNamePattern>
-          <skip>${skip.format}</skip>
-        </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>format</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
+<!--      <plugin>-->
```

Comment:
https://github.com/coveooss/fmt-maven-plugin/issues/99

not compatible with java 17 yet

Reply:
It's possible to override behaviour for specific java versions in maven:
```
<profile>
    <id>active-on-jdk-11</id>
    <activation>
        <jdk>11</jdk>
    </activation>
</profile>
```
So you can enable this plugin for all java versions prior to 17